src/Compilers/VisualBasic/Portable/Lowering/Instrumentation/DynamicAnalysisInjector.vb

```diff
@@ -128,6 +128,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return False
         End Function
 
+        Private Function GetFileIndices() As BoundExpression
```

Comment:
Fixed - I have checked to make sure that all comments in the C# version are also present in VB.